src/C#-Backend/BackendProject.csproj

```diff
@@ -0,0 +1,18 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>8cf8e960-c7d0-484d-8daa-0d02e0d74575</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GraphQL" Version="7.8.0" />
```

Comment:
@TristanFranco I'm not using this yet, how bad is it if I leave it here? I.e. Is it easy to remember later to delete unused PackageReferences?